.github/workflows/add-mmi-reviewer-and-notify.yml

```diff
@@ -0,0 +1,54 @@
+name: Add MMI reviewer and notify via Slack
+
+on:
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  process-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add reviewer when team-mmi label is added
+        if: contains(github.event.pull_request.labels.*.name, 'team-mmi')
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const github = require('@actions/github');
+            const payload = {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              pull_number: context.issue.number,
+              reviewers: ['zone-live', 'albertolive', 'shane-t']
```

Comment:
Should this be a GitHub team instead of individual users? I think you can go to `https://github.com/orgs/MetaMask/teams` and create a new team if you don't have one yet.

Reply:
Ah indeed, thank you. I've just created it, will update the PR 👍🏼